mirdata/datasets/maestro.py

```diff
@@ -147,21 +147,36 @@ def __init__(self, track_id, data_home):
 
     @utils.cached_property
     def midi(self):
-        """output type: description of output"""
+        """Description of output.
+
+        Retuns:
+            (Obj): midi"""
         return load_midi(self.midi_path)
 
     @utils.cached_property
     def notes(self):
-        """NoteData: annotated piano notes"""
+        """Annotated piano notes
+
+        Returns:
+            (NoteData): notes
+        """
         return load_notes(self.midi_path, self.midi)
 
     @property
     def audio(self):
-        """(np.ndarray, float): track's audio signal, sample rate"""
+        """Audio ground truth
```

Comment:
Audio ground truth --> The track's audio

Reply:
![Captura de pantalla 2020-12-22 a las 14 52 00](https://user-images.githubusercontent.com/16807496/102895620-4da55a80-4465-11eb-8342-24827fa44601.png)
Thanks to God. I useIntellij ^^